lumicks/pylake/channel.py

```diff
@@ -504,6 +504,172 @@ def plot(self, start=None, **kwargs):
         plt.ylabel(self.labels.get("y", "y"))
         plt.title(self.labels.get("title", "title"))
 
+    @staticmethod
+    def _annotate(start_time, annotation, annotation_direction, stop_time, color):
+        """Annotate a region or time point with some text
```

Comment:
Private methods get fully documented as well? Interesting.

Reply:
Why not? It promotes reuse if applicable. They should stay hidden when rendering the sphinxdocs.

Reply:
I just wasn't aware.

Reply:
Ah okay. Well, I don't think we are always consistent in it, but I try when I remember to do so.